src/frontend/screens/WebView/index.tsx

```diff
@@ -146,6 +146,16 @@ export default function WebView() {
     navigate('/login')
   }
 
+  const [webviewPreloadPath, setWebviewPreloadPath] = useState('')
+  useEffect(() => {
+    const fetchWebviewPreloadPath = async () => {
+      const path = await window.api.getWebviewPreloadPath()
+      setWebviewPreloadPath(path)
+    }
+
+    void fetchWebviewPreloadPath()
+  }, [])
```

Comment:
I actually wanted to use the app's preload script and just put this in the window object, but I can't properly import paths there, so I'll leave that for another time as an improvement.

Reply:
Maybe add a TODO comment so we won't forget about it